modules/installation-guide/partials/proc_importing-untrusted-tls-certificates-old.adoc

```diff
@@ -0,0 +1,310 @@
+----
+$ {orch-cli} create configmap __<config-map-name>__ --from-file=__<certificate-file-path>__ -n=__<{project-context}-namespace-name>__ -o yaml --dry-run | {orch-cli} apply -f -
+----
++
+To apply more than one certificate, add another `--from-file=_<certificate-file-path>_` option to the above command.
+
+. Configure the {prod-short} installation to use the ConfigMap:
++
+====
+For a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operators] deployment:
+
+. Edit the `spec.server.ServerTrustStoreConfigMapName` `CheCluster` Custom Resource property to match the name of the ConfigMap:
++
+[subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
+----
+$ {orch-cli} patch checluster {prod-checluster} -n __<{project-context}-namespace-name>__ --type=json -p '[{"op": "replace", "path": "/spec/server/serverTrustStoreConfigMapName", "value": "<config-map-name>"}]'
+----
++
+====
+ifeval::["{project-context}" == "che"]
++
+====
+For a {prod-short} link:https://helm.sh/[Helm Chart] deployment:
+
+. Clone the https://github.com/eclipse/che[che] project.
+. Go to the `deploy/kubernetes/helm/che` directory.
+. Update the name of the configMap {prod-short} will use, by editing the `global.tls.serverTrustStoreConfigMapName` Helm Chart property to match the created or updated ConfigMap:
++
+[subs="+quotes",options="nowrap",role=white-space-pre]
+----
+$ helm upgrade che -n che --set global.tls.serverTrustStoreConfigMapName=<config-map-name> \
+   --set global.ingressDomain=__<kubernetes-cluster-domain>__ .
+----
++
+When using Minikube to run {prod-short}, substitute _<kubernetes-cluster-domain>_ with `$(minikube ip).nip.io`.
+====
+endif::[]
+
+. Restart the {prod-short} Operator, the {prod-short} server and {identity-provider} to load the new certificates:
++
+[subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
+----
+$ kubectl rollout restart -n __<{project-context}-namespace-name>__ deployment/{prod-id-short}-operator
+$ kubectl rollout restart -n __<{project-context}-namespace-name>__ deployment/keycloak
+$ kubectl rollout restart -n __<{project-context}-namespace-name>__ deployment/{prod-id-short}
+----
++
+[NOTE]
+====
+Restarting the {prod-short} components is not required anymore on {prod-short}
+ifeval::["{project-context}" == "che"]
+v7.20.0
+endif::[]
+ifeval::["{project-context}" != "che"]
+2.5.0
+endif::[]
+and higher.
+====
+
+== Verification at the {prod-short} installation level
+
+If you added the certificates without error, the {prod-short} server starts and obtains {identity-provider} configuration over https. Otherwise here is a list of things to verify:
+
+- In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator] deployment, the `CheCluster` attribute `serverTrustStoreConfigMapName` value matches the name of the ConfigMap. Get the value using the following command :
```

Comment:
```suggestion
. In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator] deployment, the `CheCluster` attribute `serverTrustStoreConfigMapName` value matches the name of the ConfigMap. Get the value using the following command :
```